Backend/Models/Database/Medicament.cs

```diff
@@ -67,13 +67,13 @@ public Medicament(CreateMedicamentDTO dto, PharmaceuticalForm pharmaceuticalForm
             Name = dto.Name;
             ActiveSubstance = dto.ActiveSubstance;
             BarCode = dto.BarCode;
-            IsPrescriptionRequired = dto.IsPrescriptionRequired;
-            IsReimbursed = dto.IsReimbursed;
+            IsPrescriptionRequired = (bool) dto.IsPrescriptionRequired;
+            IsReimbursed = (bool) dto.IsReimbursed;
             Country = dto.Country;
-            BasePrice = dto.BasePrice;
-            Surcharge = dto.Surcharge;
-            IsSellable = dto.IsSellable;
-            ReimbursePercentage = dto.ReimbursePercentage;
+            BasePrice = (decimal) dto.BasePrice;
+            Surcharge = (double) dto.Surcharge;
+            IsSellable = (bool) dto.IsSellable;
+            ReimbursePercentage = (int) dto.ReimbursePercentage;
```

Comment:
What if casting fails? Maybe it is better to use `??` operator and provide a default value?

Reply:
Generally it should not fail as DTO is validated before this